cloud/pkg/csidriver/controllerserver.go

```diff
@@ -63,14 +63,17 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 		return nil, status.Error(codes.InvalidArgument, "Volume Capabilities missing in request")
 	}
 
-	volumeID := uuid.NewUUID().String()
+	volumeID, err := uuid.NewUUID()
```

Comment:
```suggestion
	volumeID := uuid.New().String()
```
Let's use the v4 UUID uniformly

Reply:
v4 UUID is from https://github.com/satori/go.uuid which means generate UUID based on random numbers.

Check [google uuid library](https://github.com/google/uuid/blob/master/version4.go#L9-L31), `NewUUID` bases on random numbers too.

Reply:
v1 and v4 are generated by different mechanisms, we need ro choose one of them, v4 is recommended. Ref: https://stackoverflow.com/questions/20342058/which-uuid-version-to-use

Or use uuid from k8s directly.  https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/uuid/uuid.go

Reply:
It use UUID v4 indeed, refer to https://github.com/google/uuid/blob/master/version4.go#L17.